data/json/npcs/isolated_road/isolated_road_jay_dialogue.json

```
@@ -188,31 +188,31 @@
+      "It's my house.  Cody's in the other building, if you've met her.  We dug a trench and set up some milspec turrets.  I programmed them, Cody armored them.  Now we're just trying to get by.  This ain't the wizard of Oz, you can look behind the curtain over there."
     ],
     "speaker_effect": { "effect": [ { "u_add_var": "gunsmith_mentioned_cody", "type": "dialogue", "context": "artisans", "value": "yes" } ] },
     "responses": [
-      { "text": "Where'd you get half a dozen mil spec turrets?", "topic": "TALK_GUNSMITH_TURRETS" },
+      { "text": "Where'd you get half a dozen milspec turrets?", "topic": "TALK_GUNSMITH_TURRETS" },
       { "text": "…", "topic": "TALK_GUNSMITH_SERVICES" }
     ]
   },
   {
     "id": "TALK_GUNSMITH_TURRETS",
     "type": "talk_topic",
-    "dynamic_line": [ "[staring off into the distance] The owners didn't need them anymore.  Not anymore…" ],
+    "dynamic_line": [ "*stares off into the distance.  \"The owners didn't need them anymore.  Not anymore…\"" ],
     "responses": [ { "text": "…", "topic": "TALK_GUNSMITH_SERVICES" } ]
   },
   {
     "id": "TALK_GUNSMITH_CODY",
     "type": "talk_topic",
-    "dynamic_line": [ "Stuff is fucked all over bud, how about we stick to talking business." ],
+    "dynamic_line": [ "Stuff is fucked all over, bud.  How about we stick to talking business?" ],
     "responses": [ { "text": "Alright.", "topic": "TALK_GUNSMITH_SERVICES" } ]
   },
   {
     "id": "TALK_GUNSMITH_KORD",
     "type": "talk_topic",
     "dynamic_line": [
-      "Where did you… I, I don't know what to say.  Cody put you up to this, didn't they?  The weapon is beautiful, modern rails, ergonomic furniture, recoil balancing system.  I know Cody thinks they did something wrong but all I remember is being so angry… just blindingly furious and then running.  When I woke up everything was fine.  Thank you for letting me see this and getting Cody and I talking.  If you could help me out with something for Cody I'll hook you up with a casket mag for that thing and a box of the good ammo, stuff from my reserves."
+      "Where did you… I- I don't know what to say.  Cody put you up to this, didn't she?\"  Jay takes the rifle reverently.  He turns it over so carefully that you could mistake him for holding a child instead of a gun.  \"This thing is is beautiful.  Modern rails, ergonomic furniture, recoil balancing system.  I know Cody thinks that she did something wrong, but all I remember is being so angry… just blindingly furious.  Then I was running.  When I woke up everything, was fine.\"\n\nJay hands back the Kord.  \"Thank you for letting me see this, and for getting Cody and I talking.  If you could help me out with something for Cody, I'll hook you up with a casket mag for that thing and a box of the good ammo.  Stuff from my reserves."
```

this isn't a suggestion for you since I realize it's my mistake but the character has just said Cody asked them to do it.
```suggestion
      "Where did you… I- I don't know what to say.  \"  Jay takes the rifle reverently.  He turns it over so carefully that you could mistake him for holding a child instead of a gun.  \"This thing is is beautiful.  Modern rails, ergonomic furniture, recoil balancing system.  I know Cody thinks that she did something wrong, but all I remember is being so angry… just blindingly furious.  Then I was running.  When I woke up everything, was fine.\"\n\nJay hands back the Kord.  \"Thank you for letting me see this, and for getting Cody and I talking.  If you could help me out with something for Cody, I'll hook you up with a casket mag for that thing and a box of the good ammo.  Stuff from my reserves."
```